lua/http.lua

```diff
@@ -9,9 +9,12 @@ local function make_http_request(method, uri, headers)
 
     local client_body_reader, _ = httpc:get_client_body_reader()
```

Comment:
This doesn't need to be done if you already have a body. Maybe structure this as a true if/else?

```lua
local body = ngx.var.request_body
if body ~= nil then
    local body, _ = httpc:get_client_body_reader()
end
```

That may be a premature optimization, but it also makes the code easier to read IMO

Reply:
Done